examples/with-redux-observable/lib/redux.js

```diff
@@ -0,0 +1,48 @@
+import { Provider } from 'react-redux'
+import { initializeStore } from '../store/store'
+import App from 'next/app'
+
+export const withRedux = (PageComponent) => {
+  const WithRedux = ({ initialReduxState, ...props }) => {
+    const store = initializeClientSideStore(initialReduxState)
+    return (
+      <Provider store={store}>
+        <PageComponent {...props} />
+      </Provider>
+    )
+  }
+
+  // Make sure people don't use this HOC on _app.js level
+  if (process.env.NODE_ENV !== 'production') {
+    const isAppHoc =
+      PageComponent === App || PageComponent.prototype instanceof App
+    if (isAppHoc) {
+      throw new Error('The withRedux HOC only works with PageComponents')
+    }
+  }
+
+  // Set the correct displayName in development
+  if (process.env.NODE_ENV !== 'production') {
+    const displayName =
+      PageComponent.displayName || PageComponent.name || 'Component'
+
+    WithRedux.displayName = `withRedux(${displayName})`
+  }
+  return WithRedux
+}
```

Comment:
This can be removed, right?

Reply:
Actually, this entire file looks like it could be removed

Reply:
Resolved.